community_meeting_record.yaml

```diff
@@ -0,0 +1,10 @@
+time: 2020.4.15
+host: sunshinemingo
+attendee: [sunshinemingo, wengzhenjie, gymgym1212, zhuxiangning, AnAsteria, will-ww, xiaoya-Esther, yangming, birdflyi, JingbenShi668, tedliu1]
+agenda:
+  - content: New data description
+    proposer: frank-zsy
+    participants: [sunshinemingo]
+  - content: Design of credit system
+    proposer: frank-zsy
```

Comment:
The proposer of this topi is @gymgym1212 I think

Reply:
Yes, I think that is more reasonable.